custom_components/amplipi/media_player.py

```diff
@@ -702,15 +756,19 @@ async def async_update(self):
 
         try:
             state = await self._client.get_status()
-            if self._group is not None:
-                group = next(filter(lambda z: z.id == self._id, state.groups), None)
+            if self._is_group:
+                group: Group = next(filter(lambda z: z.id == self._id, state.groups), None)
                 if not group:
                     self._last_update_successful = False
                     return
                 any_enabled_zone = next(filter(lambda z: z.id in group.zones, state.zones), None)
 
                 if any_enabled_zone is not None:
                     enabled = True
+                
+                related_zones = [state.zones[zone_index].source_id for zone_index in group.zones]
```

Comment:
since this captures the sources connected to a zone, does it make more sense to call this attached_sources or maybe connected_sources ?